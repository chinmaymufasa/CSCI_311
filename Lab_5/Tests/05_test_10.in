2
1
2
2
2
3
2
4
2
5
2
6
2
7
2
8
2
9
2
10
2
11
2
12
2
13
2
14
2
15
4
5
6
3
12
4
5
6
3
9
4
5
6
3
15
4
5
6
3
13
4
5
6
3
11
4
5
6
3
14
4
5
6
-1
