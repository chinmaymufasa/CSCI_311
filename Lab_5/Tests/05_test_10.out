8 4 2 1 3 6 5 7 12 10 9 11 14 13 15 
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 
1 3 2 5 7 6 4 9 11 10 13 15 14 12 8 
8 4 2 1 3 6 5 7 13 10 9 11 14 15 
1 2 3 4 5 6 7 8 9 10 11 13 14 15 
1 3 2 5 7 6 4 9 11 10 15 14 13 8 
8 4 2 1 3 6 5 7 13 10 11 14 15 
1 2 3 4 5 6 7 8 10 11 13 14 15 
1 3 2 5 7 6 4 11 10 15 14 13 8 
8 4 2 1 3 6 5 7 13 10 11 14 
1 2 3 4 5 6 7 8 10 11 13 14 
1 3 2 5 7 6 4 11 10 14 13 8 
8 4 2 1 3 6 5 7 11 10 14 
1 2 3 4 5 6 7 8 10 11 14 
1 3 2 5 7 6 4 10 14 11 8 
8 4 2 1 3 6 5 7 14 10 
1 2 3 4 5 6 7 8 10 14 
1 3 2 5 7 6 4 10 14 8 
6 4 2 1 3 5 8 7 10 
1 2 3 4 5 6 7 8 10 
1 3 2 5 4 7 10 8 6 
